packages/backend-deployer/src/cdk_error_mapper.test.ts

```diff
@@ -641,6 +641,13 @@ npm error enoent`,
     errorName: 'CloudFormationDeletionError',
     expectedDownstreamErrorMessage: undefined,
   },
+  {
+    errorMessage: `No valid tables found. Make sure at least one table has a primary key.`,
```

Comment:
I have 2 questions:
1. took a brief look into this error, it comes from the data team's repo, from here: https://github.com/aws-amplify/amplify-category-api/blob/e4d16911236c5751e02609c8c3b069543535883a/packages/amplify-graphql-schema-generator/src/ts-schema-generator/helpers.ts#L176 
I think we have a few other non-cdk errors that are getting mapped in this file as well, just want to confirm that this is ok.
2. You are committing some changes to package.lock. Is that intended?

Reply:
looking into this more (thanks to Kamil's comment), it does seem that we don't have tooling in place for classifying anything other than what is coming from backend-deployer

also yes it seems package.lock is being updated on clean build but i can try again